### DISCUSSION TITLE
Improving Critical Cartography and Participatory Mapping

### DISCUSSION THREAD

**Discussion by latishasumardy** [albertkun/21S-ASIAAM-191A#79] [Past Thinking Caps]
I think that participatory mapping and critical cartography are both important tools that allow people to both understand more about communities that utilize these maps and also contribute to these databases to create a more cohesive dataset that accurately reflects the people in the community. Critical cartography cannot be improved without participatory mapping, as it is important to get relevant and meaningful data in order to actually create a map that is meaningful. Some things that we can do to improve these two tools are increasing accessibility to people who may want to contribute to things like dataset or maps, but are unable to because they lack the skills to do so. There are a few ways to approach this. One way to do this is simplify the platforms being used to create these maps. This way, it becomes a lot easier for people who want to use it to learn how to create/modify maps, as they don't have to learn how to code from scratch. Another way to do this is make it more accessible for people to learn how to make these maps or contribute to databases. Although it is definitely much easier to learn things on the internet these days, with many free resources like codecademy and khan academy, it often seems very intimidating for beginners, and it deters them from actually trying. By creating more accessible resources for people, like free classes at community centers or live online classes with mentors, it will push more people to try it out and allow people to create maps that will have a meaningful impact on society.

**Comment by jason-muljadi** [comment]
I definitely agree that there needs to be more accessibility to coding so that we can have these geospatial resources to be more easily accessible. I think earlier education to coding will help, although therein lies a privileged aspect of communities that are of greater socioeconomic status has more of an opportunity to delve into this world more readily. I also wonder how to ensure greater outreach of these programs and perhaps gamify it? Making coding more exciting could potentially be a way that folks can learn as well!

**Comment by jsxu315** [comment]
I agree that accessibility to coding is really important! I feel the same way about online tutorials; they end up making me more confused and discouraged most of the time. I thought that increasing accessibility to materials that already exist could be another resource in addition to the classes you mention. I’ve also seen some really good free college courses online, like Harvard’s CS50, where there’s lots of flexibility and support for beginners and it feels more on par with an actual in-person class. I don’t know if there are many good classes available online that are directly related to the coding that people use for mapmaking, but given how many classes there are, they have to exist somewhere. 

I also wondered if incentivizing college students/community members to teach people could also help increase accessibility; it seems like a pretty big population with lots of skillsets to draw from, but it would also come with its own obstacles.
